tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -581,6 +581,17 @@ platform_tests/broadcom:
     conditions:
       - "'marvell' in asic_type"
 
+platform_tests/broadcom/test_ser.py:
+  skip:
+    reason: "platform does not support test_ser"
+    conditions:
+      - "platform in ['x86_64-cel_e1031-r0']"
+      - https://github.com/sonic-net/sonic-mgmt/issues/6218
+  skip:
```

Comment:
Not sure whether two skips in one rule work, may we try using 'OR' to combine these two skips?
for example:
platform_tests/broadcom/test_ser.py::test_ser:
  skip:
    reason: "platform does not support test_ser"
    conditions_logical_operator: 'OR'
    conditions:
      - "platform in ['x86_64-cel_e1031-r0'] and https://github.com/sonic-net/sonic-mgmt/issues/6218"
      - "asic_type not in ['broadcom']"

Reply:
thanks, I applied your proposal